core/CHANGELOG.md

```diff
@@ -1,13 +1,14 @@
 # 0.34.0 - unreleased
 
-- Introduce `StreamMuxerEvent::map_inbound_stream`. See [PR 2691].
```

Comment:
I deleted this changelog entry because it is not released yet and it no longer makes any sense if the entire type is gone now!